src/js/components/table.js

```diff
@@ -0,0 +1,220 @@
+const select = require("../utils/select");
+const behavior = require("../utils/behavior");
+const { CLICK } = require("../events");
+const { prefix: PREFIX } = require("../config");
+
+const TABLE = `.${PREFIX}-table`;
+const SORTED = "aria-sort";
+const ASCENDING = "ascending";
+const DESCENDING = "descending";
+const SORT_OVERRIDE = "data-sort-value";
+const ICON_PATH = "../../dist/img/sprite.svg";
```

Comment:
@jaredcunha Is it a bad idea to hardcode the icons path in here? i want to dynamically change the button icon based on sort state, but not sure how to do that and still have the icons be anything but background images.

Reply:
I don't think we should have the path in the JS. It's going to be hard to change if someone is using USWDS JS out of the box and decides on a different image path. There's a Sass setting for this, but not a way to update in JS that I'm aware of.